pkg/pillar/utils/cloudconfig/cloudconfig.go

```diff
@@ -0,0 +1,88 @@
+package cloudconfig
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	fileutils "github.com/lf-edge/eve/pkg/pillar/utils/file"
+	"gopkg.in/yaml.v2"
+)
+
+type CloudConfig struct {
```

Comment:
In https://github.com/lf-edge/eve/pull/3520/files you'll see yetus annotations once yetus has run. Please address the ones which are about in new/changed lines.

Reply:
Done